dev-docs/feature-conditional.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: page
+title: Conditional Ad Units - <font color="red">NEW!</font>
+description: Using labels for conditional bids
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# Conditional Bids and AdUnits
+
+Prebid 1.0 introduced the `sizeConfig` feature primarily for [responsive ad designs]({{site.baseurl}}/dev-docs/feature-responsive.html), focusing on constraining sizes based on device.
+But a number of other scenarios are supported as well:
+
+1. What if some bidders should be skipped for some devices?
+1. What if some bidders have different parameters for different devices?
+1. What if some AdUnit auctions should be skipped entirely for some devices?
+1. What if some bid requests apply only to users originating from certain countries? 
+
+By supporting these scenarios, header bidding can be more efficient - the browser can send bids to a more surgical set of bidders based on device size or other attributes the page code can create.
+
+Before going into how these scenarios work, a few words about the design of Prebid's approach. With Prebid 1.0,
+we introduce 'labels', which are just generic tags. Labels are defined by code in the page and then used by certain AdUnits and bids.
+
+The basic steps are:
+
+1. Build up an array of labels from two sources: either as an output of sizeConfig or as an optional argument to requestBids().
+1. Apply label targeting to AdUnits or specific bids.
+
+Let's jump into an example.
```

Comment:
I think this should say something like "See below for examples", followed by a Table of Contents for the page using the snippet below.  That way the reader can get an overview of the links to various use cases and jump to the one they want.  It will also make it easier to link directly to those sections in emails/Slack/etc.

```
* TOC
{:toc}
```